src/main/java/ai/knowly/langtorch/store/vectordb/integration/pgvector/schema/PGVectorValues.java

```diff
@@ -0,0 +1,19 @@
+package ai.knowly.langtorch.store.vectordb.integration.pgvector.schema;
+
+import ai.knowly.langtorch.schema.io.Metadata;
+import lombok.*;
+
+import java.util.Optional;
+
+/**
+ * Represents the values of a PGVector.
+ */
+@Data
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder(toBuilder = true, setterPrefix = "set")
+public class PGVectorValues {
+    @NonNull
+    private final String id;
+    private final float @NonNull [] values;
```

Comment:
Move this annotation to the beginning?

Reply:
It has to be like this because we want the individual items in the array to not be null. If we move the annotation to the beginning it means we are referring to the array itself as not null.